modules/ObjectControl/inc/objectcontrol.hpp

```diff
@@ -314,6 +335,18 @@ class ObjectControl : public Module
 	void disarmObjects();
 	//! \brief
 	void startScenario();
+	//! \brief Resets the test by offering a back to start trajectory. Still needs arm and start commands to execute the reset.
+	void resetTest();
+	//! \brief Reloads the scenario trajectories for each object.
+	void reloadScenarioTrajectories();
+	//! \brief Updates the paths in the GUI to reflect the new trajectories.
+	void updateTrajectoryGUI(uint32_t id);
+	//! \brief Callback for the trajectory request. Sends the new trajectory to the object.
+	void trajectoryCallback(const rclcpp::Client<atos_interfaces::srv::GetObjectTrajectory>::SharedFuture future);
+	//! \brief Callback for the return trajectory request. Sends the new trajectory to the object.
+	void returnTrajectoryCallback(const rclcpp::Client<atos_interfaces::srv::GetObjectReturnTrajectory>::SharedFuture future);
+	//! \brief Requests a new trajectory to the object depending on the current state (resetting).
```

Comment:
Requests a new trajectory? The below method is called setObjectTrajectory. Either of comment/method name does not describe what actually happens in the method.

Reply:
Updated the documentation, let me know if you're satisfied